### DISCUSSION TITLE
Paypal and Github sponsors

### DISCUSSION THREAD

**Discussion by mo8it** [tokio-rs/tokio#5492] [General] [answered]
I sponsor Tokio on Github, but since Paypal is removed as a payment method starting with tomorrow, I don't have another option for donation because I don't have a credit card. Could you please add an alternative for people that want to support with some money?

I would personally recommend Liberapay: https://liberapay.com
It is free and open source and has renewing donations :)

**Comment by Darksonn** [comment, marked as answer]
You may still be able to use paypal to donate via https://opencollective.com/tokio.
